src/app/shared/utilities/functions.ts

```diff
@@ -0,0 +1,6 @@
+export function isUpdate<T>(obj: any): obj is T {
```

Comment:
this branch was made before the header github stuff was ready so you have to manually add it here